crates/test-utils/Cargo.toml

```diff
@@ -23,6 +23,7 @@ alloy-primitives.workspace = true
 alloy-provider.workspace = true
 
 eyre.workspace = true
+futures.workspace = true
```

Comment:
note since diff from upstream

Reply:
actaully, is this needed?

Reply:
Used when spawning anvil-zksync in memory